packages/protobuf-example/package.json

```diff
@@ -6,7 +6,6 @@
     "build": "../../node_modules/typescript/bin/tsc --noEmit",
     "add": "tsx src/add.ts",
     "list": "tsx src/list.ts",
-    "pregenerate": "rm -rf src/gen",
```

Comment:
One less shell process to run, and this npm script is not portable, so I'm very happy that we can remove it.